feat: IPv6 connectivity support in NGINX One APIs (#1135)
nginx/documentation: content/nginx-one/changelog.md

Stay up-to-date with what's new and improved in the F5 NGINX One Console.

## September 16, 2025

### IPv6 endpoints for NGINX Agent and NGINX Plus usage reporting

> Your instances running in dualstack or IPv6-only environments are now able to communicate with NGINX One APIs via IPv6.

Reviewer: FYI, per our style guide, https://github.com/nginx/documentation/blob/main/documentation/style-guide.md, we avoid "latinisms". The industry seems to use "dual-stack". And I presume we're referring to NGINX One Console APIs from https://docs.nginx.com/nginx-one/api/api-reference-guide/

```suggestion
Your instances which run in dual-stack or IPv6-only environments can now communicate with NGINX One Console APIs through IPv6 addresses.
```

> Please check the [Getting Started Guide]({{< ref "/nginx-one/getting-started.md#install-nginx-agent" >}}) for IP ranges required to be enabled in your firewalls.

Reviewer 2: ```suggestion
See the [Getting Started Guide]({{< ref "/nginx-one/getting-started.md#install-nginx-agent" >}}) for the IP ranges you need to allow in your firewalls.
```
